src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -545,13 +549,17 @@ public String getBuildCompletedCommand(MemoryImprint memoryImprint, TaskListener
         } else {
             //Just as bad as failed for now.
             command = config.getGerritCmdBuildFailed();
+            // Some builds could have failed, but are already deleted and not
+            // available for score calculation.
+            // Set pessimistic upper boundary on verified value.
+            maxAllowedVerifiedValue = config.getGerritBuildFailedVerifiedValue();
```

Comment:
The only relevant branch for setting this. In other branches of this if-else statement this code would be redundant as correct verified values will be set as consequence of entry conditions permitting execution of those branches.